cmake/scripts/osx/ArchSetup.cmake

```diff
@@ -24,6 +24,13 @@ else()
   endif()
 endif()
 
+if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "arm64" AND
+   CMAKE_HOST_SYSTEM_NAME STREQUAL "Darwin")
+   set(ENABLE_XBMCHELPER OFF)
+ else()
+   set(ENABLE_XBMCHELPER TRUE)
```

Comment:
Theres no rule around it, but id avoid mixing OFF/ON TRUE/FALSE. just stick to one pair